packages/react/src/components/Link/Link.js

```diff
@@ -76,7 +77,11 @@ Link.propTypes = {
   /**
    * Specify whether you want the link to receive visited styles after the link has been clicked
    */
-  visited: PropTypes.bool,
+  visited: deprecate(
+    PropTypes.bool,
+    `\nThe prop \`visited\` for Link has been deprecated and will
+  be removed in the next major release.`
+  ),
```

Comment:
If we want users to opt in to visited link styles (meaning by default, there are no visited link style differences), then maybe we shouldn't deprecate this prop. I can see it being used to conditionally add a className that will then add purple visited link styles. i.e. Thoughts? @tw15egan @emyarod @laurenmrice

Reply:
just FYI the basis of the PR is to return to opt-out rather than opt-in styles based on the confusion it caused even our own issue triaging, as well as established research e.g. [1](https://www.nngroup.com/articles/change-the-color-of-visited-links/) [2](https://www.usability.gov/sites/default/files/documents/guidelines_book.pdf) [3](https://design-system.service.gov.uk/styles/typography/)

that being said, given that @dakahn has removed the request for review it seems this is not an a11y concern and primarily a ux one

Reply:
Yeah sorry, meant to comment. As long as contrast requirements are met and we have a logical consistency across the system with various states for links we're good from an accessibility and usability standpoint.